### ISSUE TITLE
Cart is not keyboard accessible

### ISSUE THREAD

**Issue by BuckyBuck135** [CodeStitchOfficial/Advanced-Astro-Snipcart#3]
Initial tests show that the cart is not accessible via keyboard navigation.
Investigating.
